docs/content/installation.rst

```diff
@@ -18,10 +18,11 @@ Python dependencies:
 * pyBigWig >= 0.3.16
 * hicmatrix >= 15
 * pysam >= 0.14
-* matplotlib >=3.1.1,<=3.3.2
+* matplotlib >= 3.1.1, <= 3.3.2
```

Comment:
Will be in conflict with the matplotlib version PR

Reply:
No problem. I will deal with conflict one at the time. There are so many PR that I anticipate quite a lot of conflicts.